Ome/html/Day_7/Assignment7_Q1/Home.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
Same here

Reply:
Done please check and let me know in case of any changes

Reply:
Good